shared/resources/Images.xcassets/LaunchImage.launchimage/Contents.json

```diff
@@ -1,52 +1,46 @@
 {
   "images" : [
     {
-      "orientation" : "portrait",
-      "idiom" : "iphone",
-      "extent" : "full-screen",
-      "minimum-system-version" : "8.0",
-      "filename" : "Default@2x.png",
-      "scale" : "2x"
-    },
-    {
       "extent" : "full-screen",
       "idiom" : "iphone",
-      "subtype" : "retina4",
-      "filename" : "Default-568h@2x.png",
+      "subtype" : "736h",
+      "filename" : "Default1242x2208.png",
       "minimum-system-version" : "8.0",
       "orientation" : "portrait",
-      "scale" : "2x"
+      "scale" : "3x"
     },
     {
-      "orientation" : "portrait",
-      "idiom" : "ipad",
       "extent" : "full-screen",
+      "idiom" : "iphone",
+      "subtype" : "736h",
+      "filename" : "Default2208x1242.png",
       "minimum-system-version" : "8.0",
-      "filename" : "Default-Portrait.png",
-      "scale" : "1x"
+      "orientation" : "landscape",
+      "scale" : "3x"
     },
     {
-      "orientation" : "landscape",
-      "idiom" : "ipad",
+      "orientation" : "portrait",
+      "idiom" : "iphone",
       "extent" : "full-screen",
       "minimum-system-version" : "8.0",
-      "filename" : "Default-Landscape.png",
-      "scale" : "1x"
+      "subtype" : "667h",
+      "scale" : "2x"
     },
     {
       "orientation" : "portrait",
-      "idiom" : "ipad",
+      "idiom" : "iphone",
+      "filename" : "Default640x960.png",
       "extent" : "full-screen",
-      "minimum-system-version" : "8.0",
-      "filename" : "Default-Portrait@2x.png",
+      "minimum-system-version" : "7.0",
       "scale" : "2x"
     },
     {
-      "orientation" : "landscape",
-      "idiom" : "ipad",
       "extent" : "full-screen",
-      "minimum-system-version" : "8.0",
-      "filename" : "Default-Landscape@2x.png",
+      "idiom" : "iphone",
+      "subtype" : "retina4",
+      "filename" : "Default640x1136.png",
+      "minimum-system-version" : "7.0",
```

Comment:
XCode made the change.
When I manually change it to 7.0, I get a compiler warning:
"(...)/SalesforceMobileSDK-iOS/shared/resources/Images.xcassets: An iPhone Retina (4-inch) launch image for iOS 7.0 and later is required."
Then you get the black bar for those devices.